styles/main.css

```diff
@@ -167,43 +167,37 @@ div .back-to-top:hover {
 
 main {
     display: grid;
-    grid-template-columns: 50% 50%;
+    grid-template-columns: 1fr 1fr;
     column-gap: 2em ;
 }
- .bio {
-     grid-row: 1 / 2; grid-column: 1 / span 2;
+.bio {
+    grid-column: 1 / span 2;

```

Comment:
You don't need to specify grid-row (or grid-row-start & grid-row-end, which are both implicitly set when you set grid-row) if you just want a grid item to be in its natural row.  Here, we're only interested in making sure that the item spans 2 columns, for which the grid-column declaration that you have here is enough.